packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -2212,6 +2213,87 @@ curl -v -X GET \
 ]
 ```
 
+#### List apps by name
+
+Enumerates all apps by the app name
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/apps?filter=name+eq+\"template_swa\""
+```
+
+##### Response example
+
+```json
+[
+  {
+    "id": "0oabkvBLDEKCNXBGYUAS",
+    "name": "template_swa",
```

Comment:
I'm not sure if `template_swa` is a good example. But more importantly we need this example to list at least two different apps to be clear all apps with the same name will be listed.

Reply:
@alifatolahi-okta - I added examples with 2 apps `workday` and 'cornerstone`. Notice, I used instance labels different from the app name. E.g. 

- For workday, instances labels are `hrportal1`, `hrportal2`
- For cornerstone, instance labels are `training1`, `training2`